File: test/python/runme.py
#!/QOpenSys/pkgs/bin/python3
# -*- coding:utf-8 -*-

import subprocess as subp
import json

def quote(s):
  return '\'' + s + '\''

parm_ctrl         = '*before(819)*after(819)'
parm_ctrl_ccsid   = '819'
parm_in           = json.dumps([{"cmd":{"exec": "cmd", "error": "on","value": "CHGFTPA AUTOSTART(*SAME)"}}])
parm_max_out_size = '32000'

iserv = "/usr/bin/iService " + \
          quote(parm_ctrl) + " " + \
          quote(parm_ctrl_ccsid) + " " + \
          quote(parm_in) + " " + \
          quote(parm_max_out_size)

print('** Running command: %s' %(iserv))

rc, out = subp.getstatusoutput(iserv)
if rc != 0:
  print('(E) rc = %d' %(rc))
else:
  try:
    jout = json.loads(out)
    print('STATUS = ', jout[0]['STATUS'])
  except Exception as err:
    print('(E) exception = ', str(err))

print(out)

File: test/nodejs/runme.js
const progress = require('child_process');

function quote(str)
{
  return "\'" + str + "\'";
}

let parm_ctrl         = '*before(819)*after(819)';
let parm_ctrl_ccsid   = 819;
let parm_in           = [{"cmd":{"exec": "cmd", "error": "on","value": "CHGFTPA AUTOSTART(*SAME)"}}];
let parm_max_out_size = 32000;

let run = 'iService ' 
            + quote(parm_ctrl) + ' '
            + quote(parm_ctrl_ccsid) + ' '
            + quote(JSON.stringify(parm_in)) + ' '
            + quote(parm_max_out_size);
            

console.log("Running command = \n" + run);

progress.exec(run, (err, stdout, stderr) => {
  if (err) {
    console.error("Failed to run cmd with error = \n" + err);
    return;
  }

  if (stderr) {
    console.error("Cmd failed with stderr = \n" + err);
    return;
  }

  let result = {};
  try{
    result = JSON.parse(stdout);
    console.log("Output is = \n" + JSON.stringify(result));
  } catch(e)
  {
    console.log("Invalid JSON format = \n" + stdout);
    console.log("Failed with parse output as JSON with error = \n" + e);
  }
});